tests/test_state_metrics/test_fidelity_of_separability.py

```diff
@@ -0,0 +1,62 @@
+"""Tests for State Fidelity of Seperability."""
+
+import numpy as np
+import pytest
+
+from toqito.states import bell
+from toqito.state_metrics import fidelity_of_separability
+from toqito.matrix_ops import tensor
+
+
+# test mixed rho
+mixed_rho = 1/2 * bell(0) @ bell(0).conj().T + 1 / 2 * bell(
+    3) @ bell(3).conj().T
+
+# test pure entangled rho
+u = bell(0)
+entangled_rho = u * u.conj().T
```

Comment:
Generally, one uses `@` for vector/matrix multiplication and `*` for scalar multiplication.